### PR TITLE
Add audible notifcation on broadcast error

### PR DESCRIPTION
**Pull request by weeman1337**
should be merged after https://github.com/vector-im/element-web/pull/25312

closes https://github.com/vector-im/element-web/issues/25132

PSF-1997

## Checklist

-   [x] Tests written for new code (and old code if feasible)
-   [x] Linter and other CI checks pass
-   [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
Here's what your changelog entry will look like:

## ✨ Features
 * Add audible notifcation on broadcast error ([\#10654](https://github.com/matrix-org/matrix-react-sdk/pull/10654)). Fixes vector-im/element-web#25132.

### CODE REVIEW

File: src/voice-broadcast/models/VoiceBroadcastRecording.ts [hunk 1]
```diff
@@ -333,10 +334,27 @@ export class VoiceBroadcastRecording
...
         await this.stopRecorder(false);
         this.setState("connection_error");
     }
 
+    private async playConnectionErrorAudioNotification(): Promise<void> {
+        if (localNotificationsAreSilenced(this.client)) {
+            return;
+        }
+
+        const audioElement = document.querySelector<HTMLAudioElement>("audio#errorAudio");
```
Comment (robintown) [inline src/voice-broadcast/models/VoiceBroadcastRecording.ts:349]: How does this element get into the the document?

**Review by robintown — COMMENTED** [review]
I have a question

File: src/voice-broadcast/models/VoiceBroadcastRecording.ts (hunk 1 above)
Comment (weeman1337) [reply src/voice-broadcast/models/VoiceBroadcastRecording.ts:349]: Very good question :laughing: I forgot to link the Element Web PR. Also a comment would probably be a good idea…

**Review by robintown — APPROVED** [review]
Makes sense now, thanks

**Pull request merged**